### PR TITLE
PAYARA-2965 update hk2 to 2.5.0

### PR DESCRIPTION
**Pull request by mulderbaba**

### CODE REVIEW

**Comment by mulderbaba** [comment]
jenkins test

**Comment by mulderbaba** [comment]
jenkins test

**Comment by mulderbaba** [comment]
jenkins test

**Comment by mulderbaba** [comment]
jenkins test

**Comment by mulderbaba** [comment]
jenkins test

**Comment by mulderbaba** [comment]
jenkins test

**Comment by mulderbaba** [comment]
jenkins test

File: nucleus/pom.xml [hunk 1]
```diff
@@ -289,10 +289,11 @@
 
 
         <!-- Other -->
-
-        <hk2.version>2.5.0-b61.payara-p1</hk2.version>
+        <hk2.version>2.5.0.payara-p1</hk2.version>
         <hk2.plugin.version>2.5.0-b61.payara-p2</hk2.plugin.version>
```
Comment (MattGill98) [inline nucleus/pom.xml]: Does this have a new version as well?

Comment (Cousjava) [reply nucleus/pom.xml]: The plugin version is separate from the hk2 version and are releases independently. As long as the plugin has not been changed, this does not need to be updated.

Comment (MattGill98) [reply nucleus/pom.xml]: The plugin does however have a new version, so I just wanted to check if it was appropriate to update it

**Comment by mulderbaba** [comment]
jenkins test

File: nucleus/pom.xml (hunk 1 above)
Comment (mulderbaba) [reply nucleus/pom.xml]: updated the plugins version as well.

**Pull request merged**